mdoc-interfaces/src/main/scala/mdoc/interfaces/EvaluatedWorksheet.java

```diff
@@ -15,5 +16,5 @@
   public abstract List<Path> classpath();
   public abstract List<Dependency> dependencies();
   public abstract List<Repository> repositories();
-
+  public abstract Optional<String> exportableEvaluation();
```

Comment:
My expectation is that the return value of "exportableEvaluation" would contain data structures that allow me to implement my own pretty-printing logic. The `EvaluatedWorksheet` data structure already contains this information.
```suggestion
  public abstract Optional<String> prettyPrinted();
```

Reply:
Alternative names, which I'm not sure are better: `pretty()`, `asPrettyString()`, `prettyPrint()`